### DISCUSSION TITLE
build failed on windows

### DISCUSSION THREAD

**Discussion by abhinavpatel0** [gleam-lang/gleam#3280] [General]
i just downloaded repo ran cargo update and then make install 

i got 700+  error 

   Compiling hostname v0.4.0
error[E0432]: unresolved import `lsp_types::Url`
  --> compiler-core\src\language_server\code_action.rs:11:73
   |
11 | use lsp_types::{CodeAction, CodeActionKind, CodeActionParams, TextEdit, Url};
   |                                                                         ^^^
   |                                                                         |
   |                                                                         no `Url` in the root
   |                                                                         help: a similar name exists in the module: `Uri`

error[E0432]: unresolved import `lsp_types::Url`
  --> compiler-core\src\language_server\engine.rs:23:66
   |
23 | use lsp_types::{self as lsp, Hover, HoverContents, MarkedString, Url};
   |                                                                  ^^^
   |                                                                  |
   |                                                                  no `Url` in the root
   |                                                                  help: a similar name exists in the module: `Uri`

error[E0432]: unresolved import `lsp_types::Url`
  --> compiler-core\src\language_server\server.rs:22:22
   |
22 |     Range, TextEdit, Url,
   |                      ^^^
   |                      |
   |                      no `Url` in the root
   |                      help: a similar name exists in the module: `Uri`

**Comment by lpil** [comment]
You have broken compilation by running `cargo update` and upgrading to incompatible versions. We do not support the versions you have upgraded to.

**Reply by abhinavpatel0** [reply]
i think it was an issue with nightly rust I switched to stable and its running fine

**Reply by lpil** [reply]
It's because you ran `cargo update`.

**Reply by abhinavpatel0** [reply]
yeah , thank you
